src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+html {
+  --text-color: #00acdc;
+  --header-height: 60px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  min-width: 1024px;
+  display: flex;
+  height: var(--header-height);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
```

Comment:
It's better to add padding left and right to header instead margin-left to logo and margin-right to navigation